static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -0,0 +1,283 @@
+// @flow
+import $ from 'lib/$';
+import bean from 'bean';
+import bonzo from 'bonzo';
+import fastdom from 'fastdom';
+import email from 'common/modules/email/email';
```

Comment:
I've got a conversion PR out for this module (#17951), so we just have to be careful with merge order

Reply:
yeah I saw that – I'll wait till you merge, probably easier that way?